guides/deployment/heroku.md

```diff
@@ -175,6 +175,16 @@ url: [scheme: "https", host: "mysterious-meadow-6277.herokuapp.com", port: 443],
 force_ssl: [rewrite_on: [:x_forwarded_proto]],
 ```
 
+Also, regardless if you are using distillery or mix releases, the buildpack requires that you set `server: true` so the prod.exs configuration should appear something like this.
+
+```elixir
+config :hello, Hello.Endpoint,
+  server: true,
```

Comment:
`server: true` should not be necessary on Heroku as long as the server is being started with `mix phx.server` (as `mix phx.server` does it for you). So can you please double check if Heroku is starting your app with `mix phx.server`?

Reply:
@josevalim It seems to be using `mix run --no-halt` from what I can see in the logs on Heroku. 

```
2019-11-15T18:34:52.174322+00:00 heroku[web.1]: Starting process with command `mix run --no-halt`
2019-11-15T18:34:56.399341+00:00 app[web.1]: 18:34:56.399 [info] Running GameWeb.Endpoint with cowboy 2.7.0 at :::55169 (http)
2019-11-15T18:34:56.409063+00:00 app[web.1]: 18:34:56.408 [info] Access GameWeb.Endpoint at https://elixirmatch.herokuapp.com
```

Here is the single [commit](https://github.com/toranb/workshop-deployed/commit/5b33d1e1f89d9f522a8fbafe6c6b2606ff83eba2) w/ my buildpack and mix config changes

Reply:
Then you are running into the issue described here: https://github.com/phoenixframework/phoenix/pull/3597/files#diff-58026df9e0782f005bac538b6685333eR121

You either have only one buildpack or they are in the wrong order. Using the correct order should fix it. :)